src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -237,11 +237,6 @@ NuGet.Commands.DisableSourceArgs.DisableSourceArgs() -> void
 NuGet.Commands.DisableSourceArgs.Name.get -> string
 NuGet.Commands.DisableSourceArgs.Name.set -> void
 NuGet.Commands.DisableSourceRunner
-NuGet.Commands.DownloadDependencyResolutionResult
```

Comment:
Removed `DownloadDependencyResolutionResult` type from the public api because it has no references outside the assembly.